CONTRIBUTING.md

```diff
@@ -109,6 +109,23 @@ Note the path must start with must start with a `.` to be considered local
 
 The `--verbose` flag enables additional logs which will help to identify the source of a problem. The `--reload` will clear the local cache which might have been created by a different `snowpack` version. Learn more about [Snowpack's CLI flags](https://www.snowpack.dev/#cli-flags).
 
+## Pull Request Guidelines
+
+- Checkout a topic branch from a base branch, e.g. `master`, and merge back against that branch.
+
+- If adding a new feature:
+
+  - Provide a convincing reason to add this feature.
```

Comment:
If adding a feature, it probably should have been brought up in a "discussion" instead before the PR was created. If you're having to convince anyone that the feature is a good idea at the PR stage, it's probably too late (I have closing PRs that could have been fleshed out more as discussions)